### PR TITLE
Update ADR

### PR DESCRIPTION
**Pull request by preardon**
Some Spelling mistakes and a potential ordering issue

### CODE REVIEW

File: docs/adr/0011-brighter-support-for-bulk-messaging-operations.md [hunk 1]
```diff
@@ -13,26 +13,26 @@ To support Transactional Messaging the Command Processor uses an Outbox when pro
...
 * A `ClearOutbox` call to retrieve the `Message` from the `Outbox` and dispatch it via a `Producer`
 
-Within these two steps we peform a number of activities.
+Within these two steps we perform a number of activities.
 
 For `DepositPost` we:
 
-* Lookup the `IAmAProducer` associated with a `Request
-* Translate an `IRequest` into the `Message`, excuting the `IAmAMessageTransform` and `IAmAMessagemapper` pipeline
+* Translate an `IRequest` into the `Message`, executing the `IAmAMessageTransform` and `IAmAMessageMapper` pipeline
```
Comment (iancooper) [inline docs/adr/0011-brighter-support-for-bulk-messaging-operations.md:20]: I think we look up the producer now, in order to add the Publication to the pipeline

Comment (preardon) [reply docs/adr/0011-brighter-support-for-bulk-messaging-operations.md:21]: I've added this back in

**Comment by preardon** [comment]
@iancooper  resolved your comments let me know if you're happy with the rest

**Review by iancooper — APPROVED** [review]

**Comment by iancooper** [comment]
All good.

**Pull request merged**